zerver/actions/realm_settings.py

```diff
@@ -61,6 +61,20 @@ def do_set_realm_property(
         property=name,
         value=value,
     )
+
+    message_edit_settings = [
```

Comment:
Let's add a comment that these settings have a different event format due to their history.

Reply:
Added comment.